cns/restserver/internalapi.go

```diff
@@ -12,10 +12,8 @@
 	"reflect"
 
```

Comment:
This should also pass the CRD spec details. And we should cache the complete spec.

Also as part of reconcile, ips which are listed in ToBeDeleted list must be marked as PendingRelease if they still exists in PodIpConfigState.